### PR TITLE
ship_it: Add ship_it plugin [gsoc]

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/14

### CODE REVIEW

File: plugins/ship_it.plug [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+module = ship_it
+
+[Documentation]
+description = Show motivational ship it squirrel images.
+
+[Python]
+version = 3
+
+[Errbot]
+
```
Comment (jayvdb) [inline plugins/ship_it.plug]: trailing blank line.

File: .coafile [hunk 2]
```diff
@@ -19,3 +19,7 @@ default_actions = PEP8Bear: ApplyPatchAction
 
 [all.linelength]
 bears = LineLengthBear
+
+[all.links]
+bears = InvalidLinkBear
+
```
Comment (jayvdb) [inline .coafile]: trailing blank line.

File: plugins/ship_it.plug (hunk 1 above)
Comment (meetmangukiya) [inline plugins/ship_it.plug]: This is autogen from errbot, so it is present in each of them, will remove it everywhere :+1:

File: .coafile [hunk 3]
```diff
@@ -23,4 +23,3 @@ bears = LineLengthBear
 
 [all.links]
 bears = InvalidLinkBear
-
```
Comment (Udayan12167) [inline .coafile]: Unrelated change.

Comment (meetmangukiya) [inline .coafile]: There is no such deletion :/

Comment (meetmangukiya) [inline .coafile]: I don't even see this change in files changed tab :thinking:

Comment (meetmangukiya) [inline .coafile]: also, see the change stats, its -0

Comment (Udayan12167) [inline .coafile]: Shows -1 for me.

Comment (meetmangukiya) [inline .coafile]: I see it now :+1: in commit-wise view

Comment (meetmangukiya) [inline .coafile]: done

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/58/commits/a6fb1c0d2653cafffe38797d80e2fc53f30514e3

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/58/commits/69352515a15d9c7e1384290bada3407390b86a7d

**Comment by meetmangukiya** [comment]
@rultor merge
